python/ray/remote_function.py

```diff
@@ -149,8 +151,8 @@ def invocation(args, kwargs):
                     num_return_vals)
             else:
                 object_ids = worker.core_worker.submit_task(
-                    self._function_descriptor.get_function_descriptor_list(),
-                    list_args, num_return_vals, resources)
+                    self._function_descriptor_list, list_args, num_return_vals,
```

Comment:
@pcmoritz was this the only optimization? I thought there were a couple more.

Reply:
There was a little more in https://github.com/ray-project/ray/pull/4522 concerning the argument parsing, I'll see if that's still relevant.